sdk/webpubsub/Microsoft.Azure.WebPubSub.Common/src/Internal/grrrgbbk.owj~

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What's this file? Looks like maybe a backup file something created? Probably shouldn't be checked in, or at least not with this file extension.